src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.html

```diff
@@ -14,16 +15,49 @@
         <div class="offset-md-3">
           <button class="btn btn-accent m-3"
                   name="create-cluster"
-                  [routerLink]="'/dashboard'"
                   (click)="createCluster()"
                   i18n>Create Cluster</button>
           <button class="btn btn-light"
                   name="skip-cluster-creation"
-                  [routerLink]="'/dashboard'"
                   (click)="skipClusterCreation()"
                   i18n>Skip</button>
         </div>
       </div>
     </div>
   </div>
 </div>
+
+<div class="card"
+     *ngIf="startClusterCreation">
+  <div class="card-header"
+       i18n>Create Cluster</div>
+  <div class="container-fluid">
+    <cd-wizard [stepsTitle]="stepTitles"></cd-wizard>
+    <div class="card-body vertical-line">
+      <ng-container [ngSwitch]="currentStep?.stepIndex">
+        <div *ngSwitchCase="'1'"
+             class="ml-5">
+          <h4 class="title"
+              i18n>Add Hosts</h4>
+          <br>
+          <cd-hosts [clusterCreation]="true"></cd-hosts>
+        </div>
+        <div *ngSwitchCase="'2'"
```

Comment:
No need of this here..I'm already introducing it in review section in a later PR.

Reply:
I know. but I needed it for the NextStep function where I call the host list and update the hosts array using the host list. Apparently calling everything at once in the onSubmit() function is not working as expected.